packages/dids/package.json

```diff
@@ -101,6 +102,7 @@
     "@web/test-runner-playwright": "0.11.0",
     "c8": "9.1.0",
     "chai": "5.1.1",
+    "chai-as-promised": "7.1.2",
```

Comment:
This was intentionally removed in a prior PR so that we can eventually migrate off Mocha & Chai.
```suggestion
```